jwst/master_background/tests/test_nirspec_corrections.py

```diff
@@ -59,7 +59,7 @@ def test_fs_correction():
 
 @pytest.mark.parametrize('name,status',
                          [('BKG101', True), ('bkg101', True),
-                          ('background_101', False), ('101', False),
+                          ('background_101', True), ('101', False),
```

Comment:
What required this test change?

Reply:
There were some old data in the regtest suite that were still using `background` instead of `bkg`.  Instead of changing the data, @melanieclarke who authored the original change agreed that it's fine for both `background` and `bkg` to be allowed